docs/topics/concept-getting-started-intro.adoc

```diff
@@ -0,0 +1,53 @@
+
+= Introduction to {ProductName}
+This section provides a brief overview of the foundational concepts behind {ProductName}.
+
+.Modern Application Development
+Modern application development involves building and deploying to a cloud environment such as OpenShift. This requires:
+
+* familiarity with the target cloud environment 
+* an application architecture designed to run in that cloud environment
+* design patterns that enable you to manage your services
+* developer tools that simplify the development and release cycle
+
+The goal of {ProductName} is to provide opinionated architectures, design patterns, tools, and best practices in ready-to-go examples that run in OpenShift in order to enable modern application development.
+
+
+.OpenShift
+link:https://www.openshift.com[Red Hat OpenShift] is a container application platform that brings link:https://www.redhat.com/en/topics/containers/what-is-docker[docker] and link:https://www.redhat.com/en/topics/containers/what-is-kubernetes[Kubernetes] to the enterprise. link:{link-launcher-oso}[{OpenShiftOnline}] is a public cloud hosted and supported by Red Hat. A {OpenShiftLocal} is a local cloud that you can install and run on your localhost such as the link:https://developers.redhat.com/products/cdk/overview/[{CDK}] or link:https://www.openshift.org/minishift/[{Minishift}].
+
+
+[[launcher-details]]
+.The {launcher} Application
+As a part of {ProductName}, the {launcher} application provides a hassle-free way of creating functional example applications called Missions as well as an easy way to build and deploy those Missions to OpenShift. The {launcher} application is available link:{link-launcher-oso}[directly on {OpenShiftOnline}]. You also have the option to link:{link-launcher-openshift-local-install-guide}#create-launcher-app[install and run it on your {OpenShiftLocal}].
+
+
+.Missions and Boosters
+A Mission implements a link:http://microservices.io/patterns/microservices.html[Microservice pattern] such as:
+
+* Health Check
+* REST or HTTP endpoint 
```

Comment:
"REST or HTTP" sounds weird. In theory, you REST is protocol-independent, but in practice, everyone does REST over HTTP.